README.md

```diff
@@ -104,6 +104,12 @@ The configuration for Github Actions, Azure Pipelines, Appveyor and Travis CI al
 - fix/*
 - pr/*
 
+Note that when opening a Pull Request (PR) the use of a topic branch name is not required and even discouraged as PRs are always build on CI and the use of a topic branch name will lead to duplicating all builds.
+
+A good strategy is to only use a topic branch name when actively working on a changeset and when you want CI to run branch for each push of that branch.
```

Comment:
when you want CI to _test a_ branch for each push of that branch.

Reply:
Good catch, thanks! Actually simply removing the word "branch" is shorter and more precise.